### ISSUE TITLE
Replace ce entity with one that applies to MRI and PET data equally well

### ISSUE THREAD

**Issue by tsalo** [bids-standard/bids-2-devel#46]
Currently, MRI uses `ce` (contrast enhancing agent) to indicate exogenous chemicals used to enhance contrasts, while the PET BEP, which uses tracers, plans to use the `acq` entity because `ce` doesn't reflect common terminology for PET users.

I propose that we define a new entity, perhaps called `tracer`, that would work for both technologies and would replace `ce`. 

The addition of a `tracer` entity was proposed by @mnoergaard in https://github.com/bids-standard/bids-specification/pull/633#discussion_r563794146. In the hopes of not cluttering up the specification with a new entity in BIDS 1.X, I think it would be best to replace the existing `ce` in 2.0.

**Comment by effigies** [comment]
If "tracer" works for both contexts, I would support moving to that. Difficult to shorten to 2-3 characters (`trc`, maybe?), but perhaps also unlikely to be concatenated with so many other entities that a 6 character entity would lead to unreadable filenames.

**Comment by melanieganz** [comment]
So from the PET perspective tracer would be good and shortening it to `trc` would be fine, but not great in terms of readability. But are the MR people OK to call a contrast-enhancing agent a tracer? That seems slightly misleading to me.

**Comment by tsalo** [comment]
It _would_ be good to get input from MR folks who use contrast-enhancing agents. I don't know anyone who does though.

**Comment by nicholst** [comment]
This is going to be really hard because, just linquistically, the two core terms are really different. 

For MRI injections "contrast agent" is ubiquetous in MRI and implicit with the passive nature of the injection.  In PET, the injection is the very source of the signal... a better analogy is: What is the BIDS language for the chemical species imaged (i.e. the origin of the MRI signal)? 

* [H-1] - Protons, the default
* [He-3] - Hyperpolarized 3He MRI
* [C-13] - Hyperpolarized carbon-13 MRI
* [F-19] - Fluorine MRI
* [Na-23] - Sodium MRI
* [P-31] - Phosphorus MRI

(please, someone who knows more about MRI edit the names I've assigned if I've mangled it.)

In short, I don't think we should try to harmonise this.  Contrast agent `ce` is something that modifies the MRI signal, and BIDS 2.0 for PET should create a new label, e.g. `trc`, for the origin of the PET signal.  And we should probably have yet a new keyword for chemical species of the MRI sequence.

**Comment by melanieganz** [comment]
Since we haven't finished the actual PET BEP review, we can still change the use of `acq` to `trc`. Would this then be ok for people that we make this change now?

**Comment by nicholst** [comment]
I think moving to `trc` is a good call... there isn't a perfect `acq` analogy, but if we were back in the days of retractable septa, you could imagine `acq` being used to tag 2D vs 3D acquistions.   As another example, what if you took a single list mode dataset and played it out into sets of 5 min frames, and then did it again with, say, 7 min frames... wouldn't that be a good use for `acq`?

**Comment by mnoergaard** [comment]
I would argue that this latter case would imply a new reconstruction, as the acquisition of the data has not changed, but the reconstruction into frames has. Therefore, the data would need a new ´rec´ name (recon), such as ´rec-acdynXX´ (https://bids-specification.readthedocs.io/en/bep-009/04-modality-specific-files/09-positron-emission-tomography.html). 

But I agree that ´acq´ could be used for 2D vs. 3D, even though there are only a few sites out there that still use 2D acquisitions.

**Comment by tsalo** [comment]
Thanks to everyone for responding. Apologies for the round trip on this one! I will close this and follow up in the PET BEP.
